docs/ldml/tr35.md

```diff
@@ -2235,19 +2231,21 @@ This allows for implementations that use those denormalized subtags to use the d
 The reverse operation removes fields that would be added by the first operation.
 
 1. First get max = AddLikelySubtags(inputLocale). If an error is signaled, return it.
```

Comment:
Could we also clarify what is the "it" in "return it" here? Is the "it" referring to "error" or referring to the "input string"?  For example, if the input is "qaa-CH-u-nu-latn", what should the algorithm return ?  
A. "qaa-CH-u-nu-latn"
B. "und"
C. an error